lib/urldata.h

```diff
@@ -1734,6 +1734,9 @@ struct UserDefined {
   int ftp_create_missing_dirs; /* 1 - create directories that don't exist
                                   2 - the same but also allow MKD to fail once
                                */
+  curl_sshhostkeycallback ssh_hostkeyfunc; /* hostkey check callback */
+  void *ssh_hostkeyfunc_userp;         /* custom pointer to callback */
+
```

Comment:
`#ifdef USE_LIBSSH2` so that builds without it don't "waste" space on these?

Reply:
ok